### DISCUSSION TITLE
请问resume里的“在学期间完成的相关学术成果”部分，如何给学术论文、专利、科研项目等分开编号？目前的编号是连续的，感谢

### DISCUSSION THREAD

**Discussion by HXCHOPIN** [tuna/thuthesis#949] [Q&A] [answered]
![240424](https://github.com/tuna/thuthesis/assets/29730835/9fa239d2-9304-464d-b798-5dde3810bda3)

**Comment by zepinglee** [comment, marked as answer]
1. 《写作指南》要求连续编号。
<img width="745" alt="Screenshot 2024-04-24 at 10 33 53" src="https://github.com/tuna/thuthesis/assets/12290822/6ee34e8f-2f9a-45a0-8763-db9325fff345">
2. Word 也是连续编号的。
<img width="735" alt="Screenshot 2024-04-24 at 10 34 09" src="https://github.com/tuna/thuthesis/assets/12290822/46be307c-e496-4fed-943e-57a43ed8128d">

**Reply by Harry-Chen** [reply]
同时可参见 #825

**Reply by HXCHOPIN** [reply]
好的，非常感谢两位的解答！
